liquibase/dbchangelog.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="1" author="jossuecito">
+        <!-- Dependencies table -->
+        <createTable tableName="fileurls">
+            <column name="id" type="int">
+                <constraints primaryKey="true" primaryKeyName="pk_fileurls"/>
+            </column>
+            <column name="url" type="varchar(2000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="rawdata" type="clob">-->
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addAutoIncrement
+                columnDataType="int"
+                columnName="id"
+                incrementBy="1"
+                startWith="1"
+                tableName="fileurls"/>
+
+        <createTable tableName="fileurl_childs">
+            <column name="fileurl_id" type="int">
+                 <constraints nullable="false"/>
+            </column>
+            <column name="childfileurl_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint baseTableName="fileurl_childs" baseColumnNames="fileurl_id" constraintName="fk_parent" referencedTableName="fileurls"
+                                 referencedColumnNames="id"></addForeignKeyConstraint>
+
+        <addForeignKeyConstraint baseTableName="fileurl_childs" baseColumnNames="childfileurl_id" constraintName="fk_child" referencedTableName="fileurls"
+                                 referencedColumnNames="id"></addForeignKeyConstraint>
+
+        <!-- Properties table -->
+        <createTable tableName="property">
+            <column name="property" type="varchar(100)">
```

Comment:
What's the motivating decision behind 100/1000 for these two fields?

Reply:
Pure instinct :trollface: